.github/workflows/test.yml

```diff
@@ -22,10 +22,12 @@ jobs:
           repository: saucelabs/saucectl
           path: saucectl
 
-      # Build saucectl
       - run: pushd saucectl && make build && popd
         name: Build Saucectl and run E2E tests
 
+      - name: Workaround for container permissions
```

Comment:
@FriggaHel is this because of mounting? Can we add a comment that this is required because of the docker mode we choose for running saucectl

Reply:
@vrunoa copying file has the same consequences.

Reply:
I added comment to explain why this is here.